raiden/ui/app.py

```diff
@@ -247,6 +248,31 @@ def run_app(
     known_given_network_id = given_network_id in ID_TO_NETWORKNAME
     known_node_network_id = node_network_id in ID_TO_NETWORKNAME
 
+    if config['services']['pathfinding_service_address'] is None:
+        click.secho(
+            "There is no pathfinding service defined, basic routing will be used",
+        )
+    else:
+        try:
+            pathfinding_info = requests.get(
```

Comment:
1. Please refactor this part into a function in a separate module (IMO, this should be `network/pathfinding.py`). 
2. The function should be called here. raises `InvalidPathFindingAddress` which is caught here and the respective error message is printed.
3. Once this is done, please unit test the the function for both happy case and the non-happy one.

Reply:
Thanks for the quick feedback. 
1. This new Module would then contain only that function or do you plan to expand this module in the future?
2. Just to be sure about consistency - all other CLI Arguments / external services are also checked in raiden/ui/app.py, see networkID, environmentType, etc. 
3. Will do. Could you point out where other CLI arguments are tested, such that I can reuse the code?

Reply:
The point of the feedback is to create units which are testable. At the moment almost everything in `app.py` is not properly tested. 
So:
1. I think we can also move the routing request to this module as well (not in the scope of this PR), so that all PFS functionality is grouped into this module.
2. Sure, but as i said, they're not properly tested which basically adds tech-debt that we have to deal with later on. Better start trying to prevent the tech-debt from increasing.
3. Not really, it's as simple as having a function:
```
def get_pfs_info(url: string):
    # either return the info or throw an exception
```
And in your test:
```
def test_get_pfs_info_success():
    # test happy case

def test_get_pfs_info_raises_error():
    # test that an exception was raised
```

Reply:
thanks, will do

Reply:
(1) I externalized the get_pfs_info() into a new module pathfinding.py. 
(2) If the request fails, it raises an error and stops Raiden with sys.exit(1)
(3) Added two unit tests into tests